### DISCUSSION TITLE
Create image mutation without uploading a file

### DISCUSSION THREAD

**Discussion by shardt** [saleor/saleor#6062] [Q&A] [answered]
Hi,

I would like to manage image file storage outside of saleor and update products with image URLs. Is it possible via the graphql API to create product images without uploading the file? Or is it required to actually upload a file in order to create product images?

Thanks,
Shannon

**Comment by IKarbowiak** [comment, marked as answer]
Hi @shardt,

No, it's not possible to create product images without uploading the file, uploading is required. Keep also in mind that to create a product image you need to send `multipart` request, you can find more details [here](https://github.com/jaydenseric/graphql-multipart-request-spec).

**Reply by shardt** [reply]
I guessed as much based on the documentation, but thought I would ask just to be sure. Thanks for the response.
